### ISSUE TITLE
Got bleemsync running but got a few issues...

### ISSUE THREAD

**Issue by mlpope** [pathartl/BleemSync#152]
ok, so i got bleemsync going...great...ill cut to the chase...i have quite a bit of ps1 images store on my external drive...well some of them i used to run on psp; some of those ps1 images are in the form of an eboot.pbp/keys.bin file (s)...i have a converter (popstation and etc) so i can convert them back to an iso image, then i have to use another program to convert them to bin/cue file...well i tried using magic iso to convert the iso to the bin/cue, after i converted the eboot.pbp to an iso...so i loaded the converted bin/cue on the classic and it says unsupported cd image...is it either magic iso is a crappy converter or is it that i have a ps1 image that is been changed so,many times that it is either "dirty"  or the code or something is jumbled or mixed up....will i have to open up the image with a hex editor or something...i would hate to have to re download all those files again because i had no problem when i converted them to iso and bin and burned them to play on a modded ps1 or a ps1 using swap disc method and etc.

**Comment by mlpope** [comment]
and just to add a little more info so yall can better understand...i used the bleemsync.exe and only initialized 1 game (castlevania SOTN) on a friends computer that had windows 7...but i can load that game then go to the emulator settings and options (select+ triangle) and select another cd image (i can add those on the flash drive using my computer which does not run win 7 but vista) and that has worked for me...not sure if it will be able to save on memorycard or not, but i know that i cannot use resume point for the other games i did not initialize with the .exe or whatever

**Comment by mlpope** [comment]
and just to add a little more info so yall can better understand...i used the bleemsync.exe and only initialized 1 game (castlevania SOTN) on a friends computer that had windows 7...but i can load that game then go to the emulator settings and options (select+ triangle) and select another cd image (i can add those on the flash drive using my computer which does not run win 7 but vista) and that has worked for me...not sure if it will be able to save on memorycard or not, but i know that i cannot use resume point for the other games i did not initialize with the .exe or whatever

**Comment by mlpope** [comment]
basically it seems fresh or "untainted" images i have no trouble with

**Comment by haihappenhuhaba** [comment]
I used icetea to convert the pbp files to bin files and cuemaker to get the cue files from the bin and it worked perfectly the games start normally on the classic. May that helps to fix your problem.

**Comment by chri54tkins** [comment]
What make/model/size of USB stick have you used?  All of mine have failed to boot up the machine.  Not even configuration, just go direct to the standard home screen.  The USB 2.0 drive flashes a few times on boot but then the machine ignores it.  I followed all of the config details to get the folder structure correct.  Is there a setting on the hidden menu to allow access to USB drives?  I am beginning to think that its only a few machines or a particular batch of hardware that will work.  I purchased mine in UK recently.

**Comment by Brds7t7** [comment]
I can't imagine there being a new "revised" batch already. The PS Classic has only been out for 3 weeks and hasn't been selling well. So most of the stock still in shops now are most likely still from the original run. I don't even think Sony will bother to patch it anyway, as it's probably too much effort for a low cost limited run item. They probably don't even care about people hacking it, or they would have put more effort into the security before release. Nintendo never bothered to patch any of their mini consoles either.
I know it can be quite temperamental with quite a few flash drives though, you might have to try a few.

**Comment by chri54tkins** [comment]
Well, something of a breakthrough.  I got a single game to run.  Reformatted the drive to FAT32 (not a quick format), extracted BleemSync files to the drive and put one game on it (Soul Blade).  That came up ok.  When I have tried to add others in folder 2, 3 etc it boots (flashing range and green light) but then fails.  Remove the new game folders and all is back to normal.  I wonder if its the .BIN images?  I have tried a few others that work fine with RetroPie...  Trying out a few others now.  At least I have made some progress.
